focus_areas/templates/question_template.md

```diff
@@ -1,37 +1,21 @@
-# [Area of Analysis]: [Focal Concern]
+## Description
 
-**Question:** placeholder for a variety of channels (choice?)
+## Objectives
 
+## Implementation
 
-## 1. Description
+### Filters (optional)
 
-_This section provides a rationale for why this question is important to growth maturity and decline._
-_Example for how to use this template: [Documentation](./resources/project_places-documentation.md)_
+### Visualizations (optional)
 
-## 2. Sample Objectives
+### Tools providing the metric (optional)
+- formerly: "known or sample implementations"
+- Metric must be currently deployed/available, in contrast to a tool having the "potential" to provide the metric
+- provide direct link to implementation/documentation, if applicable
 
-- _This section provides a list of reasons a community wants to assess this question_
--
+### Data Collection Strategies (optional)
 
+## Success Metrics (optional)
```

Comment:
Success Metrics should also be a sub-heading under Implementation. 

Please adjust the heading level.